.github/workflows/coverage.yml

```diff
@@ -49,7 +48,7 @@ jobs:
           source /opt/ros/foxy/setup.bash
           # for test of arci-ros (roscore, rostopic)
           source /opt/ros/noetic/setup.bash
-          cargo llvm-cov --verbose --all-features --workspace --exclude openrr-internal-codegen --doctests --lcov --output-path lcov.info
+          cargo llvm-cov --verbose --all-features --workspace --exclude openrr-internal-codegen --lcov --output-path lcov.info
```

Comment:
This drops doctests because it needs nightly. That said, it is [annoyed by a compiler bug](https://github.com/taiki-e/cargo-llvm-cov/issues/2#issuecomment-1061686546), so dropping it should not have much of an impact.